Assets/Scripts/API/DFBlock.cs

```diff
@@ -10,6 +10,7 @@
 //
 
 #region Using Statements
+using DaggerfallWorkshop.Utility.AssetInjection;
```

Comment:
That's a really good point, thanks. I did this is because I wanted to not pollute the DFBlock data class with these object processor methods used by WorldDataReplacement, but that was the wrong way to look at it. Will definitely change this.